recipes/libcurl/all/conanfile.py

```diff
@@ -111,14 +111,14 @@ def requirements(self):
             elif self.settings.os == "Windows" and self.options.with_winssl:
                 pass
             else:
-                self.requires.add("openssl/1.1.1f")
+                self.requires("openssl/1.1.1f")
```

Comment:
I am not sure what is OpenSSL update here?

Reply:
OpenSSL 1.1.1 has a new revision, without zlib. But all packages which consume OpenSSL are pointing to the old package, which has a different package id and is not available in Conan Center.

Reply:
I don't understand how does it help. you're changing `1.1.1f` to `1.1.1f`, so it effectively changes nothing - it will still use the same package with the same package id.

Reply:
its changes because the package id for 1.1.1(ANY) changed since we removed zlib from its requirements. Read this comment: https://github.com/conan-io/examples/pull/61#issuecomment-614280161

That's the situation, now we need to rebuild the packages, by server side, which is better, or do it manually by PRs.

Reply:
so, it's not needed to change anything in the recipe, just to trigger a rebuild(s)?

Reply:
There are two things here:
 * We should be able to trigger a build in the CCI to regenerate all the packages downstream without opening PRs... something to work on (too many things)
 * If we are using `semver_direct_mode`, theoretically this shouldn't happen. We need to investigate because there is a bug here (not 100% sure, but likely)

Meanwhile, it is ok to make this via PRs, but not the highest priority as it is something that will happen from time to time and we cannot be aware of it always.

Reply:
yes, this is something to be done on CI to trigger rebuilds automatically. manual PRs don't scale well, and it's not obvious how to track/detect such situations.